src/it/scala/io/iohk/ethereum/db/IodbDataSourceIntegrationSuite.scala

```diff
@@ -1,39 +1,43 @@
-package io.iohk.ethereum.mpt
+package io.iohk.ethereum.db
 
 import org.scalatest.FunSuite
 import org.scalatest.prop.PropertyChecks
 import io.iohk.ethereum.ObjectGenerators
 import io.iohk.iodb.LSMStore
 import java.io.File
 
+import akka.util.ByteString
+import io.iohk.ethereum.db.dataSource.{DataSource, IodbDataSource}
+
 class IodbDataSourceIntegrationSuite extends FunSuite
   with PropertyChecks
   with ObjectGenerators {
 
-  val KeySize: Int = 32
+  val KeySizeWithoutPrefix: Int = 32
```

Comment:
Scenarios to test:
* Insert many keys in one `recDB.update(..)` method. In `IodbDataSource insert` they are inserted separately.
* Update many keys in one `recDB.update(..)` method.
* Try to insert/update/get keys with invalid length.

Please don't implement these tests if we don't want to use IODB right now.